### PR TITLE
Feature/comms

### PR DESCRIPTION
**Pull request by afrojaA**
Added some basic code for: 

>comms() constructor
> calibrate()
>sendCommand(char type, int data)
> receivedCommand()

### CODE REVIEW

**Review by chowii — APPROVED** [review]

**Comment by chowii** [comment]
looks good. nice first one

**Review by TrideepLD — APPROVED** [review]
Looks good to me 👍

**Review by Andy2639123 — APPROVED** [review]

**Review by jayakrithi — APPROVED** [review]

**Review by safalmaharjan — APPROVED** [review]

File: MEGA/Main.ino [hunk 1]
```diff
@@ -47,15 +47,33 @@ State state();
       int baudRate = 9600;
 
     public:
+      char command;
```
Comment (natlec) [inline MEGA/Main.ino]: Might be best to make this a private variable (under `private:`) considering it'll only be accessed in the Comms class

File: MEGA/Main.ino [hunk 2]
```diff
@@ -47,15 +47,33 @@ State state();
...
       Comms() {
         Serial.begin(baudRate);
```
Comment (natlec) [inline MEGA/Main.ino]: We'll call `Serial.begin()` in the setup function once, no need to call it in the constructor for each subsystem class.

File: MEGA/Main.ino [hunk 3]
```diff
@@ -47,15 +47,33 @@ State state();
...
         Serial1.begin(baudRate);
+        
+
       }
 
       /*
         Send test command over comms and check that response is correct
       */
-      void calibrate() {
+      bool calibrate() {
```
Comment (natlec) [inline MEGA/Main.ino:61]: Will need to communicate our calibration functionality with @natleco/t1_c5

**Review by natlec — COMMENTED** [review]
Reviewed, looks good pending address of comments

**Pull request merged**